codex/blockexchange/engine/engine.nim

```diff
@@ -338,14 +398,14 @@ proc wantListHandler*(
 
     logScope:
       peer      = peerCtx.id
-      cid       = e.cid
+      # cid       = e.cid
       wantType  = $e.wantType
 
     if idx < 0: # updating entry
-      trace "Processing new want list entry", cid = e.cid
+      trace "Processing new want list entry", address = e.address
```

Comment:
you should be able to move this to the `logScope`?

Reply:
Yep, you're right. Fixing it.